services/request_service.go

```diff
@@ -110,12 +192,32 @@ func (rs RequestService) DereferenceResourceMetadata(c echo.Context) error {
 		return types.NewInvalidDIDUrlError(c.Param("did"), requestedContentType, err, true)
 	}
 	resourceId := c.Param("resource")
+
+	didMethod, _, identifier, _ := didUtils.TrySplitDID(did)
```

Comment:
And this one

Reply:
Yes, sure! Make sense! I will do it in next ticket that will be https://app.clickup.com/t/6600954/DEV-2233